yaml/landuse.yaml

```diff
@@ -332,3 +332,7 @@ filters:
   - filter: {barrier: hedge}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
     output: {kind: hedge}
+  - filter: {highway: [rest_area, services]}
+    min_zoom: GREATEST(LEAST(zoom, 16), 9)
```

Comment:
We should never see these features before zoom 11, but setting to `9` here allows them to show up earlier. (This is a problem with many of the queries!)